### DISCUSSION TITLE
Documentation at pixijselementals.com

### DISCUSSION THREAD

**Discussion by tsvetta** [pixijs/pixijs#8812] [Q&A]
Hello! Is there any way to fix typos and incorrect URLs at https://www.pixijselementals.com/?

Currently I am reading it and there are quite a few typos/errors and broken links there. I could create a PR to fix them, but unfortunatly I couldn't find any repo.

**Comment by bigtimebuddy** [comment]
That project is maintained by @miltoncandelero and he'll be able to answer this.
